travisci/python-linter_harness.sh

```diff
@@ -24,13 +24,13 @@ export MYPYPATH=$MYPYPATH:$PWD/src/python/lib
 
 PYLINT_OUTPUT_FILE=$(mktemp)
 PYLINT_ERRORS=$(mktemp)
-find "${PYTHON_SOURCE_LOCATIONS[@]}" -type f -name "*.py" -print0 | xargs -0 pylint --rcfile pylintrc --verbose | tee "$PYLINT_OUTPUT_FILE"
+find "${PYTHON_SOURCE_LOCATIONS[@]}" -type f -name "*.py" \! -name "Ortheus.py" -print0 | xargs -0 pylint --rcfile pylintrc --verbose | tee "$PYLINT_OUTPUT_FILE"
```

Comment:
Because it's Python 2. I could get pylint to run on it, but I need a different installation of pylint, so maybe another Travis stage ? I thought it was not worth it because it's the only Python 2 script